src/shared/src/logistics/redux/taskListUtils.js

```diff
@@ -1,13 +1,12 @@
 import _ from 'lodash';
 import moment from 'moment';
-import { tasksToIds } from './taskUtils';
 
 export function replaceTasksWithIds(taskList) {
```

Comment:
I guess the name `replaceTasksWithIds` is not appropriate anymore?

Reply:
Ok will be renamed to `replaceItemsWithItemIds`.